components/analog-devices/ADG1408YCPZ-REEL7.stanza

```diff
@@ -0,0 +1,94 @@
+#use-added-syntax(jitx)
+defpackage ocdb/analog-devices/ADG1408YCPZ-REEL7 :
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+
+  import ocdb/defaults
+  import ocdb/land-patterns
+  import ocdb/symbols
+  import ocdb/symbol-utils
+  import ocdb/generic-components
+  import ocdb/box-symbol
+  import ocdb/bundles
+  import ocdb/property-structs
+
+pcb-landpattern cp-16-13-adi :
+  pad p[1] : smd-pad(0.2548, 0.807999) at loc(-1.8899, 0.974999, 90.0)
+  pad p[2] : smd-pad(0.2548, 0.807999) at loc(-1.8899, 0.325001, 90.0)
+  pad p[3] : smd-pad(0.2548, 0.807999) at loc(-1.8899, -0.325001, 90.0)
+  pad p[4] : smd-pad(0.2548, 0.807999) at loc(-1.8899, -0.974999, 90.0)
+  pad p[5] : smd-pad(0.2548, 0.807999) at loc(-0.974999, -1.8899, 0.0)
+  pad p[6] : smd-pad(0.2548, 0.807999) at loc(-0.325001, -1.8899, 0.0)
+  pad p[7] : smd-pad(0.2548, 0.807999) at loc(0.325001, -1.8899, 0.0)
+  pad p[8] : smd-pad(0.2548, 0.807999) at loc(0.974999, -1.8899, 0.0)
+  pad p[9] : smd-pad(0.2548, 0.807999) at loc(1.8899, -0.974999, 90.0)
+  pad p[10] : smd-pad(0.2548, 0.807999) at loc(1.8899, -0.325001, 90.0)
+  pad p[11] : smd-pad(0.2548, 0.807999) at loc(1.8899, 0.325001, 90.0)
+  pad p[12] : smd-pad(0.2548, 0.807999) at loc(1.8899, 0.974999, 90.0)
+  pad p[13] : smd-pad(0.2548, 0.807999) at loc(0.974999, 1.8899, 0.0)
+  pad p[14] : smd-pad(0.2548, 0.807999) at loc(0.325001, 1.8899, 0.0)
+  pad p[15] : smd-pad(0.2548, 0.807999) at loc(-0.325001, 1.8899, 0.0)
+  pad p[16] : smd-pad(0.2548, 0.807999) at loc(-0.974999, 1.8899, 0.0)
+  pad p[17] : smd-pad(2.6416, 2.6416) at loc(0.0, -0.0, 0.0)
+
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(-2.1209, -2.1209), Point(-1.435139, -2.1209) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(2.1209, -2.1209), Point(2.1209, -1.435139) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(2.1209, 2.1209), Point(1.435139, 2.1209) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(-2.1209, 2.1209), Point(-2.1209, 1.435139) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(-2.1209, -1.435139), Point(-2.1209, -2.1209) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(1.435139, -2.1209), Point(2.1209, -2.1209) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(2.1209, 1.435139), Point(2.1209, 2.1209) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(-1.435139, 2.1209), Point(-2.1209, 2.1209) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(2.801899, -0.134501), Point(2.801899, -0.515501) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(2.801899, -0.515501), Point(2.547899, -0.515501) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(2.547899, -0.515501), Point(2.547899, -0.134501) ])
+  layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(2.547899, -0.134501), Point(2.801899, -0.134501) ])
```

Comment:
When discussing about large designs with @ariel-hi yesterday, I mentioned he has seen several such configurations when a rectangle is represented by 4 lines. I understand that the layer here is silkscreen and that we should consider only the outline of the rectangle.

Suggestion 1 (long term shot) : These 4 lines could be replaced by something like `Rectangle(0.254, 0.381, loc(2.674899, 0.325001, 0.0)`. Then the width (0.1524) need to be handled, which is currently not supported.

Suggestion 2: `layer(Silkscreen("F-SilkS", Top)) = Line(0.1524, [ Point(2.801899, -0.134501), Point(2.801899, -0.515501), Point(2.547899, -0.515501), Point(2.547899, -0.134501), Point(2.801899, -0.134501)])`

Reply:
Good catch. I think the guidance from Steve was to remove the outlines on land-patterns.

Reply:
Internally we use `LineRectangle` to represent this. It's defined in `utils/land-patterns.stanza`.

Reply:
imo the solution is to add representation for filled/unfilled shapes generally, but until then we can use these approximations as a workaround. Notably this doesn't work for circles that may be filled or stroked.

Reply:
I agree. That's something I was thinking about when dealing with `distance` and wanting to distinguish distance between fills and distance between outlines.